### PR TITLE
Change log updates for Aug, Sept, Oct

### PR DESCRIPTION
**Pull request by katthartic**
These updates should cover releases v0.303, v0.305, v0.306, v0.307, v0.308, v0.309, v0.310, and v0.313.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -22,10 +22,38 @@ Have a feature request, a bug report, a suggestion or an interesting use case? W
...
+## 09/28/2022
+
+### New Features
+- Introducing our public API! You can pull all your logs and quickly create and edit Fleets in bulk. Just check out Settings under _My Profile_ to get started.
+![Photo of Settings API page](./.gitbook/assets/change_log_release_v0_307_0.png)
+
+## 09/20/2022
+
+### Bug Fixes
+- Our markdown was messed up, so we got down, cleaned up, and are ready for our checkmark! Markdwon should now display as intended.
```
Comment (forstmeier) [inline docs/changelog.md:49]: Capitalize "markdown" and there's a "Markdwon" typo.

File: docs/changelog.md [hunk 2]
```diff
@@ -22,10 +22,38 @@ Have a feature request, a bug report, a suggestion or an interesting use case? W
...
+
+## 09/06/2022
+
+### Bug Fixes
+- Housekeeping did some great work but might have misplaced a few hyperlinks. They have been found and put back where they belong.
```
Comment (forstmeier) [inline docs/changelog.md:54]: :rofl:

**Review by forstmeier — CHANGES_REQUESTED** [review]
Minor tweaks.

**Comment by katthartic** [comment]
@blakeburch @ericelsken I will need approval on this from one of you to deploy. Eric - Docusaurus wanted me to do some upgrades so that is in here too.

**Review by ericelsken — APPROVED** [review]

**Comment by katthartic** [comment]
@forstmeier I need your explicit re-approval on this one.

**Review by ericelsken — APPROVED** [review]

**Review by forstmeier — APPROVED** [review]

**Pull request merged**